ui/tx/assetFlows/components/NovesSubHeadingInterpretation.tsx

```diff
@@ -61,7 +62,17 @@ const NovesSubHeadingInterpretation: FC<Props> = ({ data, isLoading }) => {
                   fontSize="lg"
                   w="fit-content"
                 />
-              ) }
+              )
+            }
+            {
+              item.address && (
+                <AddressEntity
+                  address={{ hash: item.address, is_contract: true, is_verified: true }}
```

Comment:
Not all contracts are verified. I suggest using a regular contract icon here instead of a verified one to avoid misleading the user.